### PR TITLE
nload: F2 Option Crash and Wrong adapter read, patch from http://sour…

### PR DESCRIPTION
**Pull request by samm-git**
…ceforge.net/p/nload/bugs/8/

### CODE REVIEW

File: Library/Formula/nload.rb [hunk 1]
```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
+  patch :dsrc do
+     url "http://sourceforge.net/p/nload/bugs/_discuss/thread/c9b68d8e/4a65/attachment/devreader-bsd.cpp.patch"
```
Comment (mikemcquaid) [inline Library/Formula/nload.rb]: Two-character ident here.

File: Library/Formula/nload.rb [hunk 2]
```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
+  patch :dsrc do
```
Comment (mikemcquaid) [inline Library/Formula/nload.rb]: Don't use `:dsrc` here.

File: Library/Formula/nload.rb [hunk 3]
```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
```
Comment (mikemcquaid) [inline Library/Formula/nload.rb]: Could you try to email the author about this patch? It worries me that there's been no reply for a year.

Comment (samm-git) [inline Library/Formula/nload.rb]: I wrote to the author > 10 months ago and never heard back. Also you may find that that there were no new versions for a long time and source code in the CVS is older then one in the release.

File: Library/Formula/nload.rb (hunk 2 above)
Comment (samm-git) [inline Library/Formula/nload.rb]: Should i put updated patch to the ticket?

File: Library/Formula/nload.rb [hunk 4]
```diff
@@ -11,6 +11,11 @@ class Nload < Formula
...
+  patch :p0 do
+    url "http://sourceforge.net/p/nload/bugs/_discuss/thread/c9b68d8e/4a65/attachment/devreader-bsd.cpp.patch"
```
Comment (DomT4) [inline Library/Formula/nload.rb]: Let's optimistically use `https://` here. It works, but redirects for now. The hope is one day SF will stop redirecting secure to plaintext.

File: Library/Formula/nload.rb (hunk 3 above)
Comment (mikemcquaid) [inline Library/Formula/nload.rb]: Ok, can you add that as a comment here? I think I'd like other maintainers to chime in here as we may remove this software if it's effectively unmaintained now :sob:

File: Library/Formula/nload.rb [hunk 5]
```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
+  patch :p0 do
+    url "https://sourceforge.net/p/nload/bugs/_discuss/thread/c9b68d8e/4a65/attachment/devreader-bsd.cpp.patch"
+    sha256 "19055158b72722f7dabff9890931094cac591bcc6de3e90a7f4744d28746ebc7"
+  end
```
Comment (mikemcquaid) [inline Library/Formula/nload.rb]: Also a newline after this line would be good too.

File: Library/Formula/nload.rb (hunk 3 above)
Comment (samm-git) [inline Library/Formula/nload.rb]: sorry,  the last think i was trying to do is to force removal of this tool. It works mostly fine, is tiny and very useful (IMHO).

File: Library/Formula/nload.rb (hunk 5 above)
Comment (samm-git) [inline Library/Formula/nload.rb]: Will do.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5436ab76917e4d8aff94ddb88a0f0fcefa895746. Thank you for another contribution to Homebrew @samm-git! :santa:

**Pull request closed without merging**